Dashboards first 20 lines optimization (#4084)
elastic/docs-content: explore-analyze/dashboards.md

---
navigation_title: Dashboards

> description: Visualize and share insights from your Elasticsearch data using interactive panels, charts, maps, and custom filters.

Reviewer: perhaps we use the {{es}} attribute here? Not 100% about using it in frontmatter

  - id: kibana
---

# Dashboards [dashboard]

**Dashboards** provide the primary way to visualize and share insights from your {{es}} data. Build interactive displays that combine multiple visualizations, metrics, and controls into a single view that helps you and your team monitor trends, investigate issues, and make data-driven decisions.

> A dashboard is composed of one or more **panels** that you arrange to tell your data story. Each panel can display visualizations such as charts, tables, metrics, and maps, static annotations like text or images, or specialized views for Machine Learning or Observability data.

Reviewer: ```suggestion
A dashboard is composed of one or more **panels** that you arrange to tell your data story. Each panel can display visualizations such as charts, tables, metrics, and maps, static annotations like text or images, or specialized views for {{ml}} or {{observability}} data.
```

> A dashboard is composed of one or more **panels** that you arrange to tell your data story. Each panel can display visualizations such as charts, tables, metrics, and maps, static annotations like text or images, or specialized views for {{ml}} or {{observability}} data.

Reviewer 2: ```suggestion
A dashboard is composed of one or more **panels** that you arrange to tell your data story. Each panel can display visualizations such as charts, tables, metrics, and maps, static annotations like text or images, or specialized views for {{ml}} or {{observability}} data.
```
Do we need to capitalize Observability in this case?

![Example dashboard](/explore-analyze/images/kibana-dashboard-overview.png "")

## Get started with dashboards [get-started]

New to dashboards? Start with these hands-on tutorials that walk you through creating your first dashboards using sample data:

> - [Create a simple dashboard to monitor website logs](dashboards/create-dashboard-of-panels-with-web-server-data.md): Build a dashboard with charts, metrics, and filters to analyze web traffic patterns

Reviewer 2: You might need to add a period when the list item contains a complete sentence.
RE https://stunning-adventure-qrvr1k2.pages.github.io/style-guide/formatting/#lists

> - [Create a dashboard with time series charts](dashboards/create-dashboard-of-panels-with-ecommerce-data.md): Learn to visualize sales trends and patterns over time using ecommerce data.

Reviewer 2: Spelling consistency for:
eCommerce
ecommerce

## Work with dashboards [work-with-dashboards]

Once you understand the basics, explore these common tasks:

**Build and customize dashboards**:
- [Building dashboards](dashboards/building.md): Learn the fundamentals of creating and configuring dashboards
- [Create a dashboard](dashboards/create-dashboard.md): Start with an empty dashboard and add your content
- [Add filter controls](dashboards/add-controls.md): Enable interactive filtering with options lists, range sliders, and time sliders
- [Add drilldowns](dashboards/drilldowns.md): Create interactive navigation between dashboards or to external URLs
- [Organize dashboard panels](dashboards/arrange-panels.md): Arrange panels using collapsible sections, resizing, and positioning

**Explore and interact**:

> - [Exploring dashboards](dashboards/using.md): Filter data, adjust time ranges, and interact with panels to uncover insights

Reviewer: ```suggestion
- [Explore dashboards](dashboards/using.md): Filter data, adjust time ranges, and interact with panels to uncover insights
```
This would match the sentence structure from the preceding list, but the following list using more "ing"s, so I'm not sure which way you want to go, but I think we should pick one.
